.gitignore

```diff
@@ -1,6 +1,9 @@
 # secrets
-config.yaml
-config.*.yaml
+*.yaml
+*.yml
+!config-example.yaml
+!.github/**/*.yaml
+!.github/**/*.yml
```

Comment:
I would actually prefer to not ignore all `*.yaml` files. They are generally useful to have in the repository, and  as we see `.github` also has them. The use-case for having randomly named `config.yaml` files is too exotic to justify general ignore rule. The original version already supports `config.dev.yaml`, `config.prod.yaml` etc.

Reply:
Fair enough, will revert this bit. :-)